samples/WinUI3/Properties/launchSettings.json

```diff
@@ -0,0 +1,10 @@
+{
+  "profiles": {
+    "Auth.UWP.Sample (Package)": {
```

Comment:
Can you rename these profiles to `WinUI3` ?

Reply:
Fixed